docs/openapi-user.yaml

```diff
@@ -1,170 +1,419 @@
 openapi: 3.0.3
-
 info:
-  title: Firma con IO User APIs
-  description: Any APIs called by IO App
-  version: 1.0.0
-
+  title: IO Sign - User API
```

Comment:
What do you think about using "Firma con IO - User API"?

Reply:
I agree. Thank you!

Reply:
Other references also have the name "io-sign" including the repository. For now I would leave it like this!

Reply:
In the context of IO the name "Firma con IO" is used I thought it was better to refer to this :-D